app/src/notifications/handleNotifications.js

```diff
@@ -1,9 +1,17 @@
 import slack from './providers/slack.js';
+import smtp from './providers/smtp.js';
 import formatNotification from './formatNotification.js';
+import 'dotenv/config';
 
 const handleNotifications = (monitor, run) => {
   const data = formatNotification(monitor, run);
-  slack.sendNotification(data);
+  
+  if (process.env.SLACK_WEBHOOK_URL) {
+    slack.sendNotification(data);
+  } else if (process.env.SMTP_HOST) {
+    smtp.sendNotification(data);
+  }
```

Comment:
Can we change this to two separate if statements so that the user has the choice to have both slack and email notifications setup?

Reply:
Yeah, sure. I guess I had assumed the user would pick one or the other, but we can do both